packages/components/src/actions/Toast/ShowCustom.tsx

```
@@ -41,18 +40,16 @@ export type IContextType = {
-    duration = Infinity,
-    dismissOnOverlayPress = true,
-    open,
-    onOpenChange,
-    name,
-  }: IShowToasterProps,
-  ref: ForwardedRef<IShowToasterInstance>,
-) {
+function BasicShowToaster({
+  children,
+  onClose,
+  duration = Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)